### ISSUE TITLE
Setup failed - Unsupported Device

### ISSUE THREAD

**Issue by rackrick** [DeerMaximum/Technische-Alternative-CMI#65]
## Version of the custom_component

1.8.5

## Describe the bug
Hey there...
During the setup of the integration the connect to CMI failed with an unsupported device error. The log message is pretty clear what's the problem but the reported device 31 is an CoE Endpoint configured on the CMI. Is it not possible to use the CMI and CoE Integration at the same time?

Greetings
rick 

## Debug log

```text
Logger: custom_components.ta_cmi
Source: custom_components/ta_cmi/config_flow.py:125 
Integration: Technische Alternative C.M.I. (documentation, issues) 
First occurred: 15:59:19 (1 occurrences) 
Last logged: 15:59:19

Error while communicating with a device (31): Device not supported
```

```text

2023-08-15 16:27:42.461 DEBUG (MainThread) [custom_components.ta_cmi] Sleep mode for 75 seconds to prevent rate limiting
2023-08-15 16:28:57.463 DEBUG (MainThread) [custom_components.ta_cmi] Try to fetch device type: 10
2023-08-15 16:28:57.875 DEBUG (MainThread) [custom_components.ta_cmi] Sleep mode for 75 seconds to prevent rate limiting
2023-08-15 16:30:12.877 DEBUG (MainThread) [custom_components.ta_cmi] Try to fetch available device channels: 10
2023-08-15 16:30:13.919 DEBUG (MainThread) [custom_components.ta_cmi] Sleep mode for 75 seconds to prevent rate limiting
2023-08-15 16:31:28.921 DEBUG (MainThread) [custom_components.ta_cmi] Try to fetch device type: 31
2023-08-15 16:31:28.932 WARNING (MainThread) [custom_components.ta_cmi] Error while communicating with a device (31): Device not supported
2023-08-15 16:31:28.943 DEBUG (MainThread) [custom_components.ta_cmi] Sleep mode for 75 seconds to prevent rate limiting
2023-08-15 16:32:43.945 DEBUG (MainThread) [custom_components.ta_cmi] Try to fetch device type: 50
2023-08-15 16:32:44.190 DEBUG (MainThread) [custom_components.ta_cmi] Sleep mode for 75 seconds to prevent rate limiting
2023-08-15 16:33:59.191 DEBUG (MainThread) [custom_components.ta_cmi] Try to fetch available device channels: 50
2023-08-15 16:34:14.393 ERROR (MainThread) [frontend.js.latest.202308020] :0:0 Script error.
```

**Comment by DeerMaximum** [comment]
Can you finish setting up the integration and the other devices? If so, you can ignore the error message, since the device (31) is only communicated with in the setup process and the error does not occur afterwards. 

The message is only a hint why some devices are not shown on the list for selection. In your case, it is the CoE node that is not supported by the API.

**Comment by rackrick** [comment]
Well... i couldn't finish setting up the integration as I wrote this issue.
Now I've tried a couple more time and made it to next dialog where I can choose the devices.

The dialoag shows an "communication error with one device".

I selected then my controller and added it but had no entities available.  Proabably because I deselected the checkbox for all channels.
On the next try I could add my controller with entites but I am missing the valve output which are two outputs paired.

Can I edit the channels after adding a device?
